library/entropy_poll.c

```diff
@@ -18,9 +18,15 @@
  */
 
 #if defined(__linux__)
+#if !defined(_GNU_SOURCE)
 /* Ensure that syscall() is available even when compiling with -std=c99 */
```

Comment:
I think this comment should be updated to something like "Ensure GNU extensions (syscall()/getrandom()) are available even when compiling with -std=c99".